mysql-plugin/docs/Mysql-batchsink.md

```diff
@@ -72,9 +72,9 @@ Data Types Mapping
 | TINYINT                        | int                   |                                                         |
 | BOOL, BOOLEAN                  | boolean               |                                                         |
 | SMALLINT                       | int                   |                                                         |
-| MEDIUMINT                      | double                |                                                         |
+| MEDIUMINT                      | int                   | Users can manually set output schema to map it to Long. |
```

Comment:
what does this comment mean? There is no output schema for a sink.

Reply:
There is similar existing comment for YEAR. Seems like this table was just copied from the source? For a sink I would expect to find what CDAP types can be written to which MySQL types

Reply:
I will work on this comment in the next set of changes, proposing to take this after this PR submission.